css/styles.css

```diff
@@ -1,50 +1,171 @@
+
 * {
-    box-sizing: border-box; /* content-box */
+  box-sizing: border-box;
 }
-*, *::before, *:after{ 
-box-sizing: inherit;
+
+.topnav {
+  background-color: #333;
+  overflow: hidden;
 }
 
-body {
-background-color: burlywood;
+.topnav a {
+  float: left;
+  color: #f2f2f2;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
 
+.topnav a:hover {
+  background-color: #ddd;
+  color: black;
 }
 
+.topnav a.active {
+  background-color:turquoise;
+  color: white;
+}
 
-h1 {
-    text-align: center;
-    font-size: 70px;
+body{
+  background-color: grey;
+  color:ivory;
+  height: 100%;
+  margin: 0;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  line-height: 1;
 }
 
-.links {
-text-align: center;
-font-size: 15px;
+h1, 
+h2, 
+h3 {
+  margin: 0;
+  font-weight: 400;
+}
 
+.lg-heading {
+  color: white;
+  font-size: 6rem;
 }
 
+.sm-heading {
+  color: white;
+  margin-bottom: 2rem;
+  padding: 0.2rem 1rem;
+  filter: grayscale();
+}
+
+a {
+  color: white;
+  text-decoration: none;
+  padding: 0.4rem;
+}
+
+.text-secondary {
+color:turquoise;
+}
+
+main {
+  padding: 4rem;
+  min-height: cal(100vh -60px);
+}
+
+.icons {
+    margin-top: 1rem;
+}
+
+.icons a:hover {
+  background-color: turquoise;
+  transition: all 1,5s;
```

Comment:
This will not have an effect as commas are not working. Should be: `transition: all 1.5s`